docs/content/en/_index.md

```diff
@@ -53,3 +55,6 @@ figure src="/images/dialects.svg" link="/images/dialects.svg" %}}
 TODO(b/287634511): add list of partner companies and universities
 
 The HEIR codebase and documentation are maintained by Google.
+
+Logo design by [Edward Chen](https://edwjchen.com/) and
+[Erin Park](https://www.linkedin.com/in/erin-park-b8a590237/)
```

Comment:
just found out erin has a portfolio website! could we change the link to: https://www.yerinstudio.com/

Reply:
Fixed, thanks!